tests/PullRequest.py

```diff
@@ -31,6 +31,8 @@
 
 import datetime
 
+import github
```

Comment:
github is always injected into the test classes, you can drop this import.

Reply:
Removing this line causes the test cases in this file to fail. See https://github.com/PyGithub/PyGithub/actions/runs/698739235